common/test/acceptance/pages/lms/learner_profile.py

```diff
@@ -165,3 +167,93 @@ def age_limit_message_present(self):
         """
         self.wait_for_ajax()
         return self.q(css='#u-field-message-account_privacy').visible
+
+    @property
+    def profile_has_default_image(self):
+        """
+        Return list of visible fields.
+        """
+        self.wait_for_field('image')
+        default_links = self.q(css='.image-frame').attrs('src')
+        return 'default-profile' in default_links[0] if default_links else False
+
+    def mouse_hover(self, element):
```

Comment:
I'd move this to a superclass as this is very useful functionality that is not specific to learner profiles.